project/OSGi.scala

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka
+
+import com.typesafe.sbt.osgi.OsgiKeys
+import com.typesafe.sbt.osgi.SbtOsgi._
+import sbt.Keys._
+import sbt._
+
+object OSGi {
+
+  // The included osgiSettings that creates bundles also publish the jar files
```

Comment:
with `The included osgiSettings` you mean the upstream defaults from the sbt-osgi plugin? But we modify it to go with the other jars? OK I guess.

Reply:
The OSGi.scala file was mainly restored to the state before support for OSGi was dropped. This comment was part of the original content. I understand it that the default settings of the OSGi plugin are adapted to generate jars with an OSGi manifest in the default location.